config/webhook/kustomization.yaml

```diff
@@ -1,6 +1,42 @@
+namespace: capi-webhook-system
+
```

Comment:
Need to add namePrefix something to the resources for the webhook as they will live in a shared namespace with other providers.

i.e. service with name `webhook-service` doesn't identify it as the CAPA service.

Reply:
Already merged in https://github.com/kubernetes-sigs/cluster-api-provider-aws/pull/1577. I have merged master into this PR to get all the required bits including the `namePrefix`.